### PR TITLE
Updated npmignore (ignore .github folder)

### PR DESCRIPTION
**Pull request by dsebastien**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ testapp/
 website/
 
 .gitattributes
+.github
```
Comment (juliemr) [inline .npmignore]: This should end with a `/` since it's a directory

**Comment by cnishina** [comment]
added the `/` with commit b28c6c1e7cc88d69bb5188d69988a42b68cd2ac0.

**Comment by dsebastien** [comment]
Thanks, sorry I didn't see the mail notification :)

**Comment by juliemr** [comment]
I reviewed this while sleepy and busy with conference stuff, I apologize. It should not have been merged - we WANT the `.github` folder for including our issue templates. I will push a revert.

**Comment by juliemr** [comment]
Aaand I'm wrong again, this was for .npmignore. I should really not try to review things within like 7 days of travel. Sorry for the noise.

**Comment by dsebastien** [comment]
No worries ;-)
Great work for the conference btw, I'm still busy watching the videos

**Pull request closed without merging**